client/src/components/_common/Form/FormField.js

```diff
@@ -14,7 +14,44 @@ import { useField } from 'formik';
 import PropTypes from 'prop-types';
 import './FormField.scss';
 
+/** A limited-choice wrapper for `FormField` */
+const FormFieldWrapper = ({ children, type }) => {
+  let wrapper;
+
+  switch (type) {
+    case 'InputGroup':
+      wrapper = <InputGroup>{children}</InputGroup>;
+      break;
+
+    case 'FormGroup':
+    default:
+      wrapper = <FormGroup>{children}</FormGroup>;
+  }
+
+  return wrapper;
+};
+FormFieldWrapper.propTypes = {
+  /** The content for the wrapper */
+  children: PropTypes.node.isRequired,
+  /** Which wrapper to use */
+  type: PropTypes.oneOf(['InputGroup', 'FormGroup'])
```

Comment:
__Console Error__

A console error (about component prop validation) appears because of a mistake in my code from PR # from my code in #312 (dd58ceb).

```suggestion
  type: PropTypes.oneOf(['InputGroup', 'FormGroup', ''])
```

_This change allows empty value to be passed, as intended (no error; component will still default to `FormGroup`)._